Source/inv.cpp

```diff
@@ -2063,9 +2063,14 @@ void UseStaffCharge(int pnum)
 
 BOOL UseStaff()
 {
+

```

Comment:
Close, but you added an empty line to the function.

Does your git tool not have a preview function? Previewing before you add `#ifdef` should help you see what parts where changed and need `#ifdef`

Reply:
Ok. I removed the newlines.